project/frontend/src/components/RestaurantCard.js

```diff
@@ -159,9 +159,20 @@ function RestaurantCard(props) {
     height: collapsed ? '0px' : imageSliderMaxHeight.current + 'px',
     transition: 'height 0.75s cubic-bezier(0.35, 0.91, 0.33, 0.97)',
   };
-  const photoUrls = details
-    ? details.result.photos.map((photo) => photo.getUrl({ maxWidth: 344 }))
-    : ['https://pixelpapa.com/wp-content/uploads/2018/11/26.gif'];
+
+  let photoUrls;
+  if (details) {
+    if (details.result.photos) {
+      photoUrls = details.result.photos.map((photo) =>
+        photo.getUrl({ maxWidth: 344 })
+      );
+    } else {
+      // TODO: Display a better "no images found" type image
+      photoUrls = ['https://pixelpapa.com/wp-content/uploads/2018/11/26.gif'];
+    }
+  } else {
+    photoUrls = ['https://pixelpapa.com/wp-content/uploads/2018/11/26.gif'];
```

Comment:
lets make the 'https://pixel...' part a const named string above maybe like placeholderImageUrl = 'http://pixel..' 
and use it here